Fix Android build

jlkalberer opened the pull request:
# Summary

Fixing the issues outlined in https://github.com/react-native-community/art/issues/64

* Updated to new version of React and fixed Android build errors
* Updated iOS podfile to build/test iOS
* Upgraded `yarn.lock`

## Test Plan
Ran the example project on iOS and Android
![Screen Shot 2020-12-03 at 8 28 39 AM](https://user-images.githubusercontent.com/1107396/101059280-f0dc1180-3542-11eb-8cd7-96426c2d00fa.png)


### What's required for testing (prerequisites)?
1. Clone the repo
2. run `yarn android`

### What are the steps to reproduce (after prerequisites)?
Verify that the example no longer crashes

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅      |
| Android |    ✅     |

## Checklist

<!-- Check completed item, when applicable, via: [X] -->

- [X] I have tested this on a device and a simulator
- [X ] I added the documentation in `README.md`
- [???] I mentioned this change in `CHANGELOG.md`
- [X] I updated the typed files (TS and Flow)
- [X ] I added a sample use of the API in the example project (`example/App.js`)

jlkalberer committed:
Fix Android build
Upgrade yarn.lock
Upgrade RN to 0.62.2

jlkalberer committed:
Fix Android build
Upgrade yarn.lock
Upgrade RN to 0.62.2

Clonescody commented:
Hi guys, when can we expect this to be merged ? I need this fix in my app.

Thanks !

Clonescody commented:
@EvanBacon Sorry to bother you, but could you check this PR really quick ? 
I feel like this issue is impacting a lot of other librairies, and I need it tbh :D 

Thanks !

jlkalberer commented:
Yeah, it's weird that they don't fix it themselves or approve this pull request :/

Commit:
fix(ios): Xcode12 build run on device crash (#70)

Since the ARTShadow structure holds a color object, it will cause a zombie object after destruction.

dependabot[bot] committed:
Bump lodash from 4.17.15 to 4.17.19 (#67)

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.15 to 4.17.19.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.19)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump acorn from 5.7.3 to 5.7.4 (#58)

Bumps [acorn](https://github.com/acornjs/acorn) from 5.7.3 to 5.7.4.
- [Release notes](https://github.com/acornjs/acorn/releases)
- [Commits](https://github.com/acornjs/acorn/compare/5.7.3...5.7.4)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Esemesek commented:
Hey, thanks for the PR. Everything looks good, but it seems that I cannot push to your master branch to fix conflicts. I'll merge this PR when conflicts will be resolved.

n0490b commented:
@jlkalberer Any ETA on fixing the merge conflicts? Thanks

jlkalberer commented:
> @jlkalberer Any ETA on fixing the merge conflicts? Thanks

I’m away from a computer until next week. I’m glad this issue is finally getting traction but this PR has been out forever...

jlkalberer commented:
I'll have a fix for this today with iOS testing.
Be ready to merge :D

jlkalberer committed:
Reverted changes made to color conversion
Tested in iOS and Android

jlkalberer committed:
Merge branch 'master' of https://github.com/Brewskey/art

jlkalberer commented:
I rebased my changes on top of this branch and have tested things again. I've updated the original PR with the new test plan.
